### PR TITLE
upgpkg: rocm-hip-{libraries,runtime,sdk} 5.1.0-1

### PR DESCRIPTION
**Pull request by JP-Ellis**

### CODE REVIEW

File: rocm-hip-runtime/PKGBUILD [hunk 1]
```diff
@@ -10,10 +10,14 @@ depends=('rocm-core' 'rocm-language-runtime' 'rocminfo' 'hip-runtime-amd' 'rocm-
...
+
+_magic_bit=36
 
 package() {
 	mkdir -p "${pkgdir}/opt/rocm/.info"
 	echo "${pkgver}-${pkgrel}-$_magic_bit" > "${pkgdir}/opt/rocm/.info/version-hiprt"
 }
+
+
+
```
Comment (acxz) [inline rocm-hip-runtime/PKGBUILD:23]: nit: see newlines throughout this PR

**Review by acxz — COMMENTED** [review]
srry for being a bit picky with the newlines

your code so beautiful that this is the only thing i can point out :stuck_out_tongue_winking_eye:

File: rocm-hip-runtime/PKGBUILD (hunk 1 above)
Comment (JP-Ellis) [reply rocm-hip-runtime/PKGBUILD]: Fixed

**Comment by JP-Ellis** [comment]
I have pushed an additional commit to make use of the upstream deb.  Feel free to squash them.

**Pull request merged**